### DISCUSSION TITLE
How to work with designers

### DISCUSSION THREAD

**Discussion by riskers** [chakra-ui/chakra-ui#8514] [Q&A] [answered]
I don't understand, how do you configure the output theme with the designer?

Our company designers use Figma, whose component colors are always different, so when I use chakra, it is very troublesome.

My code is a lot like that:

```tsx
<Button bg="aaa" color="bbb">

<Button bg="ccc" color="ddd">
```

How do I abstract the theme?

**Comment by veloware** [comment]
Have a look at the docs, there is sections on how to customise the theme, including tokens, and how to use the theme values and tokens in your components.

There is also a section on how to define a component theme so you wouldn't be passing many styles in via props

https://v2.chakra-ui.com/docs/styled-system/customize-theme

**Comment by segunadebayo** [comment, marked as answer]
Put designer colors in tokens and recipes. Do not set `bg` and `color` on every button.

https://chakra-ui.com/docs/theming/overview
